collectors/0/pxc-collector.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+"""
```

Comment:
Thanks for using env, please put the standard copyright header there. 

Also confirm that you have signed the OpenTSDB Contributor Agreement (https://docs.google.com/spreadsheet/embeddedform?formkey=dFNiOFROLXJBbFBmMkQtb1hNMWhUUnc6MQ)